### DISCUSSION TITLE
SILAC library based on heavy channel

### DISCUSSION THREAD

**Discussion by HugoGagnon** [vdemichev/DiaNN#604] [General]
Hi,
My understanding is that if we do in silico library using this setting:

--fixed-mod SILAC,0.0,KR,label
It will light peptide as a reference and then during data integration 
--channels SILAC,L,0.0,KR;SILAC,H,8.014199:10.008269,KR
will look for both channel.
Is this right?
Is there a way to use the heavy channel as reference instead. Our experimental design (IP-MS) uses the light channel for background control and should not contain key proteins into it.
We tried :

--fixed-mod SILAC,8.014199,K
--fixed-mod SILAC,10.008269,R
--original-mods
--strip-unknown-mods

But we only get the 1st fixed mod argument in the library there after.

Best,
Hugo

**Comment by vdemichev** [comment]
Hi Hugo,

DIA-NN does not have a notion of a 'reference' channel, just not applicable to how it processes things.

Can you please share the full log, including for the library generation? If the report does not match your expectations, what specifically is missing?

Best,
Vadim
